### PR TITLE
feat: populate internal path with content type field [CU-2jtv8xj]

### PR DESCRIPTION
**Pull request by ltsNotMike**
## Ticket

#152 

## Summary

You can connect the path to attributes of a related content type. When configured path (if empty) will be populated with the attribute from the selected related. Many attributes can be connected to the path. It will fall back to the next one if the previous field was empty in the related entity. 
 
## Test Plan

- Create project 
- Configure path default param on the Settings page 
- Try to add a new element with the configured content type
- The empty internal path should be populated with fields from the selected content type entity

### CODE REVIEW

**Comment by cyp3rius** [comment]
Task linked: [CU-2jtv8xj Link content type field to path](https://app.clickup.com/t/2jtv8xj)

**Review by kamilszewczyk0 — COMMENTED** [review]
For me looks fine

File: admin/src/pages/SettingsPage/index.tsx [hunk 1]
```diff
@@ -338,7 +339,7 @@ const SettingsPage = () => {
                                           placeholder={getMessage('pages.settings.form.nameField.placeholder')}
                                           onClear={() => null}
```
Comment (AndrzejBugdol) [inline admin/src/pages/SettingsPage/index.tsx:340]: Repeated few times in this file, could be improved.

**Review by AndrzejBugdol — COMMENTED** [review]
One small improvement, rest looks gooood ;)

**Comment by cyp3rius** [comment]
@ltsNotMike resolve conflicts please and we will merge it

**Pull request merged**